examples/cloudwatch-logs/variables.tf

```diff
@@ -109,7 +109,7 @@ variable "custom_s3_bucket" {
 }
 
 variable "create_secret" {
-  description = "Set to False In case you want to use SSM with your secret that contains coralogix private key"
+  description = "Set to False In case you want to use SM with your secret that contains coralogix private key"
```

Comment:
Set to False In case you want to use secrets manager with a predefine secret that was already created and contains Coralogix send your logs key